### PR TITLE
docs(*): Update for Explore Profiles

### PR DESCRIPTION
**Pull request by knylander-grafana**
### ✨ Description

Updates to the Explore Profiles doc to address changes in the UI for the public preview release.

**Related issue(s):**

Fixes: https://github.com/grafana/pyroscope-squad/issues/238

### 📖 Summary of the changes

* Screenshots need to be updated
* Old screenshots will be commented out

### CODE REVIEW

File: docs/sources/choose-a-view/index.md [hunk 1]
```diff
@@ -44,7 +48,7 @@ Determining which view to use frames the rest of your investigation.
    </td>
   </tr>
   <tr>
-   <td>All services
+   <td>Services
```
Comment (grafakus) [inline docs/sources/choose-a-view/index.md:51]:
```suggestion
   <td>All services
```

File: docs/sources/choose-a-view/index.md [hunk 2]
```diff
@@ -124,18 +128,34 @@ Determining which view to use frames the rest of your investigation.
 
 The **All services** view is the default view when you first open Explore Profiles.
 
-![All services](../images/explore-profiles-homescreen.png)
+![The Services view](../images/explore-profiles-homescreen.png)
```
Comment (grafakus) [inline docs/sources/choose-a-view/index.md:131]:
```suggestion
![The All services view](../images/explore-profiles-homescreen.png)
```

File: docs/sources/determine-use-case/index.md [hunk 3]
```diff
@@ -11,32 +11,52 @@ weight: 400
...
+For more information, refer to [Profiling types](../concepts/#profile-types/) for help selecting a profile type to match your use case.
+Refer to [Understand profile types](https://grafana.com/docs/pyroscope/latest/view-and-analyze-profile-data/profiling-types/) to learn about profile types and instrumentation methods.
 
 After you’ve identified the problem process or service, you can filter and explore using labels and flame graphs to view lower levels.
+With capabilities like the Flame graph AI interpreter or the GitHub integration, Explore Profiles helps you locate the root cause and how to address it.
+
+## Use case 1: Investigate an issue
 
-## Use case 1: Know the problem service, not the cause
+Profiling data is idea when you know there is a specific service or area where there is a performance issue.
```
Comment (grafakus) [inline docs/sources/determine-use-case/index.md:38]:
```suggestion
Profiling data is ideal when you know there is a specific service or area where there is a performance issue.
```

File: docs/sources/determine-use-case/index.md [hunk 4]
```diff
@@ -11,32 +11,52 @@ weight: 400
...
+
+If you know the affected service (use case 1), your investigation starts by viewing that service using the **All services** view and reviewing the profiles for that service.
 In the first scenario, you may have identified that the `checkoutservice` has an issue.
-You can select `checkoutservice` and then use the **Profile types** view to examine all profiles for that service.
+You can use **Search services** to locate `checkoutservice` and then use the **Profile types** view to examine all profiles for that service.
 
-## Use case 2: Know there is an issue, need to investigate
+### Example: Know there is an issue, need to investigate
 
 If you only know there is an issue and have to investigate, then your investigation starts by using the **All services** exploration type.
```
Comment (grafakus) [inline docs/sources/determine-use-case/index.md:50]: We should stick to "view" and not "exploration type" as the user will understand better IMO.

File: docs/sources/determine-use-case/index.md [hunk 5]
```diff
@@ -11,32 +11,52 @@ weight: 400
...
 Using the **Profile type** selector, you can check the services’ CPU processes, memory allocation, blocks, locks, exceptions, and other available profile types.
 
-After you locate the profile with a spike, select either **Profiles** to change to **Profile types** view and examine all profile types for that service, or select **Labels** to view the labels (such as `hostname` or `span_name`) for that service.
+After you locate the profile with a spike, select either **Profile types** view to examine all profile types for that service, or select **Labels** to view the labels (such as `hostname` or `span_name`) for that service.
+
+## Use case 2: Research performance and resource bottlenecks
+
+Illuminate performance issues across a number of dimensions when you are doing a proactive analysis.
+This could be a cost-cutting exercise or attempt to improve latency or memory usage in a non-incident scenario. This exercise could be across multiple services or within a service across profile types, labels, or a combination of the two.
+The flow guides you through identifying the bottleneck.
```
Comment (grafakus) [inline docs/sources/determine-use-case/index.md:59]: Which flow?

File: docs/sources/choose-a-view/index.md [hunk 6]
```diff
@@ -124,18 +128,34 @@ Determining which view to use frames the rest of your investigation.
...
 ### Profile types
 
 The Profile types shows one chart for each profile available for the selected service.
+You can select to display the charts in a grid or one chart per row.
 
 ![Profile types](../images/explore-profiles-profile-types.png)
 
 ### Labels
 
+The Labels view lets you navigate and analyze performance data through tags and labels.
```
Comment (grafakus) [inline docs/sources/choose-a-view/index.md:142]: Aren't tags and labels the same thing? Maybe we can just use "labels", wdyt?

File: docs/sources/determine-use-case/index.md [hunk 7]
```diff
@@ -11,32 +11,52 @@ weight: 400
...
 
-## Use case 1: Know the problem service, not the cause
+Profiling data is idea when you know there is a specific service or area where there is a performance issue.
+Maybe an alert from spike in CPU led you to profiles or maybe your logs showed OOM issue for a particular service and you need to debug it.
+Explore Profiles lets you quickly drill into a service and identify the performance issue in these scenarios.
 
-If you know the affected service (use case 1), your investigation starts by viewing that service using the **Services** view and reviewing the profiles for that service.
+### Example: Know the problem service, not the cause
+
+If you know the affected service (use case 1), your investigation starts by viewing that service using the **All services** view and reviewing the profiles for that service.
```
Comment (grafakus) [inline docs/sources/determine-use-case/index.md:44]:
```suggestion
If you know the affected service, your investigation starts by viewing that service using the **All services** view and reviewing the profiles for that service.
```

File: docs/sources/investigate/index.md [hunk 8]
```diff
@@ -34,7 +36,7 @@ When you use Explore Profiles, your investigations follow these steps.
    - From the Labels view: Select **Add to filters** from one of the areas of interest.
 1. Optional: Click and drag on any graph to select a specific time frame or data range.
 1. Use the **Labels** view to select two processes to compare.
-   - Select the checkbox in two graphs. Select **Compare** to view your selections in the **Comparison diff view**.
-   - Use the **Service** and **Profile** drop-downs to change the service or profile type displayed.
+   - Select the checkbox in two graphs. Select **Compare** to view your selections in the **Diff flame graph** view.
```
Comment (grafakus) [inline docs/sources/investigate/index.md:39]:
```suggestion
   - Select both the "Baseline" and "Comparison checkboxes. Click on the **Compare** button to view your selections in the **Diff flame graph** view.
```

File: docs/sources/investigate/index.md [hunk 9]
```diff
@@ -34,7 +36,7 @@ When you use Explore Profiles, your investigations follow these steps.
...
+   - Use the **Service** and **Profile type** drop-downs to change the service or profile type displayed.
```
Comment (grafakus) [inline docs/sources/investigate/index.md:40]:
```suggestion
   - Select both a time range in the baseline graph and in the comparison graph to view the diff flame graph.
```

**Review by grafakus — COMMENTED** [review]
Thank you for the update! Left a few comments, let me know if they make sense.

File: docs/sources/choose-a-view/index.md (hunk 6 above)
Comment (knylander-grafana) [reply docs/sources/choose-a-view/index.md:142]: Happy to update. Have we removed tags from the UI? I think using tags and labels is a holdover from where we talked about the Tag view, etc.

File: docs/sources/determine-use-case/index.md (hunk 5 above)
Comment (knylander-grafana) [reply docs/sources/determine-use-case/index.md:59]: Let's remove the last sentence in that paragraph. We'll eventually write another use case to cover the cost-cutting scenario, but not right now. 
```suggestion
```

File: docs/sources/investigate/index.md (hunk 9 above)
Comment (knylander-grafana) [reply docs/sources/investigate/index.md:40]: Thank you for this update

File: docs/sources/determine-use-case/index.md [hunk 10]
```diff
@@ -11,32 +11,52 @@ weight: 400
...
 In the first scenario, you may have identified that the `checkoutservice` has an issue.
```
Comment (knylander-grafana) [inline docs/sources/determine-use-case/index.md:45]:
```suggestion
In this example, you may have identified that the `checkoutservice` has an issue.
```

File: docs/sources/determine-use-case/index.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (knylander-grafana) [reply docs/sources/determine-use-case/index.md:44]: I went back and forth on this. Which one would apply for more people in this situation? Is it better to point them to the Search service area and then go to Profile types or to start with Profile types and select a service from the drop-down? 

What if we add a sentence below to help explain.

Comment (knylander-grafana) [reply docs/sources/determine-use-case/index.md:44]: I edited the suggestion to remove the specific example of using All services. Below, we have two sentences that provide methods for viewing the affected service.

File: docs/sources/determine-use-case/index.md [hunk 11]
```diff
@@ -11,32 +11,52 @@ weight: 400
...
-You can select `checkoutservice` and then use the **Profile types** view to examine all profiles for that service.
+You can use **Search services** to locate `checkoutservice` and then use the **Profile types** view to examine all profiles for that service.
```
Comment (knylander-grafana) [inline docs/sources/determine-use-case/index.md:46]:
```suggestion
You can use **Search services** to locate `checkoutservice` and then use the **Profile types** view to examine all profiles for that service.

Alternatively, you can select the **Profile types** view and then choose the `checkoutservice` from the **Service** drop-down list.
```

File: docs/sources/choose-a-view/index.md [hunk 12]
```diff
@@ -89,7 +93,7 @@ Determining which view to use frames the rest of your investigation.
    </td>
   </tr>
   <tr>
-   <td>Flame graphs
+   <td>Flame graph
```
[the remark this answers is not in the document]
Comment (knylander-grafana) [reply docs/sources/choose-a-view/index.md]: Done.

File: docs/sources/determine-use-case/index.md [hunk 13]
```diff
@@ -11,32 +11,52 @@ weight: 400
...
-Selecting different profile types lets you focus on memory allocation, CPU processes, allocation sizes, blocks, or blocks.
+Selecting different profile types lets you focus on memory allocation, CPU processes, allocation sizes, blocks, or mutually exclusive (mutex).
+
 The profile types available depend on how you've instrumented you app to generate your profiling data.
-For more information, refer to [Understand profile types](https://grafana.com/docs/pyroscope/latest/view-and-analyze-profile-data/profiling-types/).
+For more information, refer to [Profiling types](../concepts/#profile-types/) for help selecting a profile type to match your use case.
+Refer to [Understand profile types](https://grafana.com/docs/pyroscope/latest/view-and-analyze-profile-data/profiling-types/) to learn about profile types and instrumentation methods.
 
 After you’ve identified the problem process or service, you can filter and explore using labels and flame graphs to view lower levels.
+With capabilities like the Flame graph AI interpreter or the GitHub integration, Explore Profiles helps you locate the root cause and how to address it.
```
[the remark this answers is not in the document]
Comment (knylander-grafana) [reply docs/sources/determine-use-case/index.md:34]: Good catch. I'll add it.

File: docs/sources/determine-use-case/index.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (knylander-grafana) [reply docs/sources/determine-use-case/index.md]: I added another sentence below. See what you think.

File: docs/sources/determine-use-case/index.md (hunk 13 above)
[the remark this answers is not in the document]
Comment (knylander-grafana) [reply docs/sources/determine-use-case/index.md]: Done.

File: docs/sources/choose-a-view/index.md (hunk 6 above)
Comment (knylander-grafana) [reply docs/sources/choose-a-view/index.md]: Removed the word tag in most locations in favor of labels.

File: docs/sources/investigate/index.md (hunk 8 above)
Comment (grafakus) [reply docs/sources/investigate/index.md]: Is this suggestion correct?

File: docs/sources/investigate/index.md (hunk 9 above)
Comment (grafakus) [reply docs/sources/investigate/index.md]: Same here: is this suggestion correct?

Comment (knylander-grafana) [reply docs/sources/investigate/index.md]: Resolving since we discussed this in Slack.

**Review by grafakus — APPROVED** [review]
Thank you!

**Pull request merged**